cloud/services/computes/droplets_test.go

```diff
@@ -0,0 +1,989 @@
+/*
+Copyright 2021 The Kubernetes Authors.
```

Comment:
this is a new file the header should be 

```suggestion
Copyright 2023 The Kubernetes Authors.
```